app/next-client-app/api/scanreports.ts

```diff
@@ -137,18 +137,34 @@ export async function getScanReportTable(id: string): Promise<ScanReportTable> {
   }
 }
 
-export async function getScanReportField(
-  id: string | null
-): Promise<ScanReportField | null> {
-  if (id === null) {
-    return null;
-  }
-
+export async function getScanReportField(id: string): Promise<ScanReportField> {
   try {
     return await request<ScanReportField>(fetchKeys.fieldName(id));
   } catch (error) {
-    console.warn("Failed to fetch data.");
-    return null;
+    console.warn("Failed to fetch data. Passed ID could be null");
```

Comment:
If you're failing to retrieve data, why are we not erroring or redirecting? Creating a large object with empty strings, nulls and 0's doesn't feel like the right approach here.

Reply:
Also, specifically in this case why are we returning a big object with "default" values? Is this supposed to be an empty form with no details? If so, do we need to really set the page to not display data?

Reply:
Just to follow up - chatted with Tien (and Jon) about this - it looks like this follows a pattern already established in the app, so its beyond the remit of this PR to refactor, however the point still stands:

The API functions that hit the back end and then return an object should be neutral - they either find the object and return it, or it does not (e.g. a null). It is then up to specific pages that use that method how to handle that response. It feels strange to go to the trouble of building an "empty" object that is populated by the default values. If a particular view needs it, it should do that itself. 
In this case, the relevant view did not need the object prepopulating, and could easily handle a null (it was for a drop down, and to specific which value was selected).

Worth putting on the backlog as a bit of technical debt.